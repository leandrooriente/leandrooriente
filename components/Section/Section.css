.wrapper {
  border-top: 5px solid var(--light-gray);
  padding-top: 50px;
  margin: 50px 0 100px;
}