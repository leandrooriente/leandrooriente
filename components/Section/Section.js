import React from 'react';
import PropTypes from 'prop-types';
import css from './Section.css';

const Section = ({ children }) => (
  <section className={css.wrapper}>
    { children }
  </section>
);

Section.propTypes = {
  children: PropTypes.node.isRequired,
};

export default Section;
