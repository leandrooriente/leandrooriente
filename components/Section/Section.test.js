import React from 'react';
import { mount } from 'enzyme';
import Section from './Section';

describe('<Section />', () => {
  it('should render <Section /> component', () => {
    const wrapper = mount(
      <Section>
        Test
      </Section>
    );

    expect(wrapper).toMatchSnapshot();
  });
});
