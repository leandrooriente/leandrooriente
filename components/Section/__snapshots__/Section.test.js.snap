// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Section /> should render <Section /> component 1`] = `
<Section>
  <section
    className="wrapper"
  >
    Test
  </section>
</Section>
`;
